packages/cli/src/lib/experimental/startBackendExperimental.ts

```diff
@@ -97,9 +97,20 @@ export async function startBackendExperimental(options: BackendServeOptions) {
       .slice(['node', 'backstage-cli', 'package', 'start'].length)
       .filter(arg => !optionArgs.includes(arg));
 
+    const additionalArgs = new Array<string>();
+    if (process.env.ENABLE_NODE_WATCH && envEnv.NODE_ENV === 'development') {
+      additionalArgs.push('--watch');
```

Comment:
For more context, the approach was based on the following comment: https://github.com/privatenumber/tsx/issues/246#issuecomment-1636755059 . 
Maybe this would be helpful, but backstage is version 1.23.3, and tsx is 4.7.0